modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -243,7 +243,7 @@ $ oc patch Network.config.openshift.io cluster \
 $ oc patch Network.config.openshift.io cluster \
   --type='merge' --patch '{
     "spec": {
-      "clusterNetwork": [
+      {"clusterNetwork": [
```

Comment:
I am not sure if this change fixes anything, the previous version actually looks correct if I count the braces properly?

Reply:
Yeah, 7 can't be right. Can we close this bug as `Not a bug` in your opinion, @npinaeva ?

Reply:
I would say you should ask bug reporter about that, maybe they meant something else